.github/workflows/ci.yml

```diff
@@ -22,3 +23,30 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+  action:
+    name: GitHub Action Dry Run
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout GitHub Action
+        uses: actions/checkout@v2.4.0
+        with:
+          path: rdme-repo
```

Comment:
Why supply this instead of relying on `uses: .` where you call the validate and openapi commands? So you aren't checking out `readmeio/oas-examples` into the root dir?

Reply:
Kinda hard to explain... so when working on this, I saw a glaring issue when testing this in [an external repo](https://github.com/kanadgupta/metrotransit-nextrip-oas) that I didn't catch when testing this in the `rdme` codebase itself.

Basically [the `npm install` step](https://github.com/readmeio/rdme/blob/1f8ce22973039a1029cd0042ab43df6ec32ac7ae/action.yml#L20-L23) was being executed in the external codebase after the checkout step and not in the `rdme` codebase where it should be running.

I figured by moving the `rdme` into a subdirectory, we can ensure that it's being run the same way it would be in any other codebase.